storage/sis-netcdf/src/main/java/org/apache/sis/internal/netcdf/Resources.properties

```diff
@@ -45,3 +45,4 @@ UnexpectedDimensionForVariable_4  = Variable \u201c{1}\u201d in file \u201c{0}\u
 UnknownProjectionParameters_2     = Unknown projection parameters in file \u201c{0}\u201d: {1}
 UnmappedDimensions_4              = Variable \u201c{1}\u201d in file \u201c{0}\u201d has {2,number} dimensions but only {3,number} can be associated to a coordinate reference system.
 UnsupportedDataType_3             = NetCDF file \u201c{0}\u201d uses unsupported data type {2} for variable \u201c{1}\u201d.
+NoSupportedAxisForDimension_2     = Cannot find any valid axis for dimension {0} in file {1}.
```

Comment:
For consistency with other messages, {0} and {1} should be between quotes if their values are expected to be strings. Note that quotes use Unicode characters are different in English and French, so I suggest to do a copy-and-paste from another line.